src/Select/Select.jsx

```diff
@@ -105,6 +111,8 @@ Select.propTypes = {
   placeholder: React.PropTypes.string,
   readOnly: React.PropTypes.bool,
   searchable: React.PropTypes.bool,
+  creatable: React.PropTypes.bool,
+  creatableProps: React.PropTypes.object,
```

Comment:
this makes it a little difficult to discover what props are available when using a creatable (and also leaks the internals of this component - wrt the fact that it's currently dependent on react-select)

i'd recommend listing out explicit creatable props that we want to support in the top-level `Select.propTypes` and passing them through explicitly when we use the `Creatable` component. wdyt?